poly-synth/src/poly-synth.js

```diff
@@ -103,15 +115,47 @@ class PolySynth {
     let reverbInputNode = new GainNode(this.context_);
     let reverbOutputNode = this.createReverbGraph_(reverbInputNode);
 
-    // The client is responsible for connecting |this.output| to a destination.
-    this.output = new GainNode(this.context_);
-
-    this.voiceOutput_.connect(bitcrusherInputNode);
+    this.voiceOutput_.connect(noisegateInputNode);
+    noisegateOutputNode.connect(bitcrusherInputNode);
```

Comment:
Doesn't matter. It's a design choice. However, you have to clearly indicate the flow of the signal somehow. Otherwise, user will have no clue at all. How about a diagram in SVG?